include/xeus/xkernel.hpp

```diff
@@ -51,15 +52,17 @@ namespace xeus
                 logger_ptr logger = nullptr,
                 server_builder sbuilder = make_xserver,
                 debugger_builder dbuilder = make_null_debugger,
-                nl::json::error_handler_t eh = nl::json::error_handler_t::strict);
+                nl::json::error_handler_t eh = nl::json::error_handler_t::strict,
+                nl::json debugger_config = nl::json::object());
 
         xkernel(const std::string& user_name,
                 interpreter_ptr interpreter,
                 history_manager_ptr history_manager = make_in_memory_history_manager(),
                 logger_ptr logger = nullptr,
                 server_builder sbuilder = make_xserver,
                 debugger_builder dbuilder = make_null_debugger,
-                nl::json::error_handler_t eh = nl::json::error_handler_t::strict);
+                nl::json::error_handler_t eh = nl::json::error_handler_t::strict,
+                nl::json debugger_config = nl::json::object());
```

Comment:
Nitpicking: what do you think of inverting the last arguments so that debugger related arguments are together? Or even better, gather them in a small structure.

Reply:
ooookkkaaayy